cartridge/graphql.lua

```diff
@@ -34,8 +35,13 @@ local function get_fields()
     return vars.graphql_schema_fields
 end
 
-local function funcall_wrap(fun_name)
+local function funcall_wrap(fun_name, operation, field_name)
+    checks('string', 'string', 'string')
     return function(...)
+        for trigger, _ in pairs(vars.on_resolve_triggers) do
```

Comment:
I think using an array is redundant

Reply:
This interface is native for Tarantool. I could agree with you in this particular case, but I don't want to invent new trigger style.

Reply:
Hm, no. It's a bit not native.

- In your case setting the trigger is idempotent:
```
tarantool> box.space.log:before_replace(x)
---
- 'function: 0x010ce83f78'
...

tarantool> box.space.log:before_replace(x)
---
- 'function: 0x010ce83f78'
...

tarantool> box.space.log:before_replace()
---
- - 'function: 0x010ce83f78'
  - 'function: 0x010ce83f78'
...
```

- You don't give any guarantees about order of the execution

Reply:
But I agree that ability to setup several triggers could be useful